lib/modules/ipfs/embarkjs/name.js

```diff
@@ -0,0 +1,43 @@
+/*global __embarkIPFS*/
+
+const NoConnectionError = 'No IPFS connection. Please ensure to call Embark.Names.setProvider()';
```

Comment:
You say IPFS, but Embark.Names. Which one is it?

Reply:
In order to connect to publish to IPNS, you need to connect to an IPFS node